frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -306,11 +322,9 @@ function DashboardInner(props: DashboardProps) {
     async (dashboardId: DashboardId) => {
       initialize({ clearCache: !isNavigatingBackToDashboard });
 
-      loadDashboardParams();
```

Comment:
It was never necessary - from my testing it never actually did anything that DashboardControls wasn't already doing (i.e. parsing hash options and acting upon those values). With the new logic it's still unnecessary since we just use `useEffect` to keep track of hash parameters

Reply:
(resolving this comment for now as this is addressed)